src/main/java/mate/academy/rickandmorty/mapper/CharacterMapper.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.rickandmorty.mapper;
+
+import mate.academy.rickandmorty.config.MapperConfig;
+import mate.academy.rickandmorty.dto.external.CharacterDataDto;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.model.Character;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface CharacterMapper {
+    CharacterDto toDto(Character character);
+
+    @Mapping(source = "id", target = "externalId")
+    Character toModel(CharacterDataDto characterDataDto);

```

Comment:
See other comments related to these changes

```suggestion
    Character toModel(CharacterDataDto characterDataDto);

    List<Character> toCharacterList(List<CharacterExternalDto> list);
```